packages/react-components/react-datepicker-compat/src/components/DatePicker/DatePicker.types.ts

```diff
@@ -12,25 +11,7 @@ export type DatePickerSlots = {
   popupSurface?: Slot<'div'>;
 };
 
-// eslint-disable-next-line @typescript-eslint/naming-convention
-export interface IDatePicker {
-  /** Sets focus to the input */
-  focus(): void;
-
-  /** Reset the state of the picker to the default */
-  reset(): void;
-
-  /** Open the datepicker popup */
-  showDatePickerPopup(): void;
-}
-
```

Comment:
reset can be done though the input ref and showDatePickerPopup can be done through the open/defaultOpen/onOpenChange props.